clients/drcachesim/CMakeLists.txt

```diff
@@ -136,6 +136,7 @@ endif()
 
```

Comment:
I found the PR description not very clear: please reword to make it clear that this adding lz4 as a supported file format for final trace files (it is already used for raw trace files).

Please change the title: "migration on lz4" is not grammatically correct at least for American English.  I would s/migration on lz4/use lz4 for final trace files/.

Also in the description I don't know what "drmemtrace.crypto.aes" and the other similar strings means: is that naming an application or benchmark or something with a "drmemtrace." prefix for some reason?